ios/native/NativeProxy.mm

```diff
@@ -369,6 +372,17 @@ static CFTimeInterval calculateTimestampWithSlowAnimations(CFTimeInterval curren
     }
     return nil;
   }];
+#ifdef DEBUG
+  [animationsManager setHasDuplicateSharedTagBlock:^(UIView *view, NSNumber *_Nonnull viewTag) {
+    UIView *screen = [REAScreensHelper getScreenForView:(UIView *)view];
+    auto screenTag = [screen.reactTag intValue];
+    bool hasDuplicateSharedTag =
+        nativeReanimatedModule->layoutAnimationsManager().hasDuplicateSharedTag([viewTag intValue], screenTag);
+    if (hasDuplicateSharedTag)
+      NSLog(@"Found duplicate shared tag.");
```

Comment:
I think that we don't need this log

Reply:
Then the whole function could actually be a void since we don't need the return value 🤔